source/FpsConfig.h

```diff
@@ -85,7 +85,11 @@ class PlayerConfig {
 	float			movementRestrictionZ = 5.0f;				///< Player's restricted movement span along Z
 	bool			restrictedMovementEnabled = false;			///< If true, player's movement will be restricted along X and Z
 	bool			counterStrafing = false;					///< If true, counter strafing will be enabled. (defaults to false.)
-
+	bool			propagatePlayerConfigsToAll = false;		///< If true, all players config from the server will be broadcast to clients
```

Comment:
This seems like server-specific configuration and should likely not be added to the generic `PlayerConfig` but rather a server-specific config file.

Reply:
I'll keep this to be implemented in the next iteration. Currently, it will take a lot of work to refactor all of the code.